docs/API.md

```diff
@@ -317,16 +317,16 @@ This is useful in combination with another helper that expects a higher-order co
 ```js
 // `hasLoaded()` is a function that returns whether or not the component
 // has all the props it needs
-const spinnerWhileLoading = hasLoaded =>
+const spinnerWhileLoading = isLoading =>
   branch(
-    props => !hasLoaded(props),
+    isLoading,
     renderComponent(Spinner) // `Spinner` is a React component
   )
 
 // Now use the `spinnerWhileLoading()` helper to add a loading spinner to any
 // base component
 const enhance = spinnerWhileLoading(
-  props => props.title && props.author && props.content
+  props => !props.title && !props.author && !props.content
```

Comment:
`||` instead of `&&`
in your case will return `false` if one of fields will be defined

Also IMO better to not overuse !
and write `!(a && b)` instead of `!a || !b`